internal/backend/backendbase/base_test.go

```diff
@@ -239,3 +239,54 @@ func TestBase_nullCrash(t *testing.T) {
 		}
 	})
 }
+
+func TestBase_emptyStringsUsed(t *testing.T) {
```

Comment:
I've realised this overlaps somewhat with `TestSDKLikeApplyEnvDefaults`, but maybe it's useful to have this as a more explicit regression test?